### ISSUE TITLE
Emulator in datastore mode doesn't return "transaction" for lookup requests

### ISSUE THREAD

**Issue by onurdialpad** [firebase/firebase-tools#7359]
### [REQUIRED] Environment info

**firebase-tools: 13.11.3**

**Platform: macOS**

### [REQUIRED] Test case
```
import requests

emulator_url = 'http://127.0.0.1:54510/v1/projects/testbed-test:lookup'

payload = b'{"keys": [{"partitionId": {"projectId": "testbed-test", "namespaceId": ""}, "path": [{"kind": "TestEntity", "id": "5348024557502464"}]}], "readOptions": {"newTransaction": {"readWrite": {}}}}'

headers = {'Content-Length': '191', 'Content-Type': 'application/json'}

# POST request to the emulator
resp = requests.post(emulator_url, headers=headers, data=payload)

# check response's data
data = resp.json()

# data must include "transaction" because payload has "readOptions": {"newTransaction": {"readWrite": {}}}, but it doesn't include the "transaction".

print(data)
```

### [REQUIRED] Steps to reproduce

Described above

### [REQUIRED] Expected behavior

"data" must contain "transaction" (datastore emulator returns it as expected) according to the reference page https://cloud.google.com/datastore/docs/reference/data/rest/v1/projects/lookup

### [REQUIRED] Actual behavior
it doesn't include "transaction"

**Comment by aalej** [comment]
Thanks for the detailed report @onurdialpad! I’m able to reproduce the issue using the information provided. I’ll raise this to our engineering team so that they can take a look.

**Comment by onurdialpad** [comment]
closing this one as it has been fixed

**Comment by onurdialpad** [comment]
@harshyyy21 can you confirm if fix has been released, if so upgrading `firebase-tools` is enough or we should upgrade both gcloud and firebase-tools ?

**Comment by onurdialpad** [comment]
reopening the issue for more visibility

**Comment by harshyyy21** [comment]
Hi @onurdialpad! Apologies for the delay. The fix has been submitted but we are running into some issues cutting a new release. Please allow me this week to investigate and get a new release out as soon as possible.

**Comment by harshyyy21** [comment]
Hi @onurdialpad, we're still trying to track down what exactly is blocking our release. At the moment, it seems like it may not be related to Firestore. Please allow me to update this bug once we have a root cause. Thank you for your patience!

**Comment by harshyyy21** [comment]
Fix available in v1.19.8: https://cloud.google.com/sdk/docs/release-notes#cloud_firestore_emulator. Please re-open if there are any follow up questions or issues.
